recipes-connectivity/iot-companion/iot-companion/iot-companion.sh

```diff
@@ -0,0 +1,652 @@
+#!/usr/bin/env bash
+
+#set -Eeuo pipefail
+set -Eeo pipefail
+
+trap cleanup SIGINT SIGTERM ERR EXIT
+
+
+cleanup() {
+  trap - SIGINT SIGTERM ERR EXIT
+  # script cleanup here
+}
+
+# VERSION TAG
+version="v1.0.2"
```

Comment:
The script version does not match the recipe version. Please check which one is correct.